examples/usage/strings.ipynb

```diff
@@ -263,8 +263,8 @@
       "print(nengo.builder.operator.Reset(sig, tag=\"tag\"))\n",
       "print(nengo.builder.operator.Copy(sig, sig))\n",
       "print(nengo.builder.operator.Copy(sig, sig, tag=\"tag\"))\n",
-      "print(nengo.builder.operator.SlicedCopy(sig, sig, Ellipsis, slice(0, 1)))\n",
-      "print(nengo.builder.operator.SlicedCopy(sig, sig, Ellipsis, slice(0, 1), tag=\"tag\"))\n",
+      "print(nengo.builder.operator.Copy(sig, sig, [0], slice(0, 1)))\n",
+      "print(nengo.builder.operator.Copy(sig, sig, [0], slice(0, 1), tag=\"tag\"))\n",
```

Comment:
Why did you switch to `[0]` instead of `None`? Because there is no print with a `Copy` operator using integer list indexing?